src/Tizen.Messaging/Tizen.Messaging.Messages/Message.cs

```diff
@@ -131,6 +131,21 @@ private void AddAddress(MessagesAddress address)
             }
         }
 
+        public int GetMessageCount(int mbox, int messageType)
+        {
+            int count;
+            Log.Error(Globals.LogTag, "GetMessageCount is Called");
```

Comment:
In my opinion, Log.Error should only be used if an exception appears, not as a standard logging behaviour.